DevOps.Status/wwwroot/css/site.css

```diff
@@ -69,3 +69,8 @@ body {
   white-space: nowrap;
   line-height: 60px; /* Vertically center the text there */
 }
+
+/* Allow tooltips to be wider than the default bootstrap version */
+.tooltip-inner {
+    max-width: 400px;
+}
```

Comment:
I think you've mad a critical mistake here. You have now identified yourself as someone who knows significantly more about bootstrap than I do. So I now know who to bother when I have questions :smile: